modules/ossm-validate-smcp-kiali.adoc

```diff
@@ -0,0 +1,52 @@
+[id="ossm-validate-control-plane-kiali_{context}"]
+= Validating your SMCP installation with Kiali
+
+You can use the Kiali console to validate your {ProductShortName} installation. The Kiali console offers several ways to validate your {ProductShortName} components are deployed and configured properly.
+
+.Procedure
+
+. Log in to the {product-title} web console as a user with cluster-admin rights. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
+
+. Navigate to *Networking* -> *Routes*.
+
+. On the *Routes* page, select the control plane project, for example `istio-system`, from the *Namespace* menu.
++
+The *Location* column displays the linked address for each route.
++
+. If necessary, use the filter to find the route for the Kiali console. Click the route *Location* to launch the console.
+
+. Click *Log In With OpenShift*.
++
+When you first log in to the Kiali Console, you see the *Overview* page which displays all the namespaces in your service mesh that you have permission to view.  When there are multiple namespaces shown on the *Overview* page, Kiali shows namespaces with health or validation problems first.
++
+.Kiali Overview page
+image::ossm-kiali-overview.png[Kiali Overview page showing istio-system]
++
+If you are validating the console installation and namespaces have not yet been added to the mesh, there might not be any data to display other than `istio-system`. The tile for each namespace displays the number of labels, the *Istio Config* health, the number of and *Applications* health, and *Traffic* for the namespace.
+
+. To view the topology, click the *Graph* page, select the *Namespace* where you installed the `ServiceMeshControlPlane` from the menu, for example `istio-system`.
+
+.. If necessary, click *Display idle nodes*.
+
+.. To learn more about the *Graph* page, click the *Graph tour* link.
+
+. To view the list of applications in the `istio-system` namespace, click the *Applications* page. Kiali displays the health of the applications.
+
+.. Hover your mouse over the information icon to view any additional information noted in the *Details* column.
+
+. To view the list of workloads in the `istio-system` namespace, click the *Workloads* page. Kiali displays the health of the workloads.
+
+.. Hover your mouse over the information icon to view any additional information noted in the *Details* column.
+
+. To view the list of services in the `istio-system` namespace, click the *Services* page. Kiali displays the health of the services and of the configurations.
+
+.. Hover your mouse over the information icon to view any additional information noted in the *Details* column.
+
+. To view a list of the Istio Configuration objects in the `istio-system` namespace, click the *Istio Config* page. Kiali displays the health of the configuration.
+
+.. If there are configuration errors, click the row and Kiali opens the configuration file with the error highlighted.
```

Comment:
Note that the "istio-system" namespace or any other where the control plane is installed has a special treatment in Kiali, so it also links with the "control plane" mesh charts defined in Grafana:
![image](https://user-images.githubusercontent.com/1662329/160140347-69249cb4-ec9d-45b9-a61c-1f03d54b9561.png)

It can also help to validate that the SMCP installs the integration with Grafana correctly.